### DISCUSSION TITLE
Device drivers for 3D Trakstar Ascension 2 position sensor? (Windows 10)

### DISCUSSION THREAD

**Discussion by annap2277** [PlusToolkit/PlusLib#1165] [Q&A]
Hello,

Does the Plus app need to have the USB driver from the manufacturer also installed separately to work with the Ascension Trakstar 2 sensor? Also, does anybody know where to get the USB driver if so? The manufacturer (Ascension) seems to have changed ownership to NDI and they don't have the drivers available on their website anymore.

Thanks!

**Comment by adamrankin** [comment]
Have your reached out to NDIs support team? They are usually quite helpful

https://support.ndigital.com/s/contactsupport

We don't have the driver though, sorry.

**Comment by lassoan** [comment]
The best is to ask NDI support. If that is not feasible then you can ask on the Slicer forum (https://discourse.slicer.org) as you can reach a larger audience and there, and it is also possible to reply in private (which is useful because people may hesitate publicly sharing non-publicly-distributed files).
